param_test_node:
  ros__parameters:
    test_param: "abcdddd"
